frontend/lib/src/components/shared/Tooltip/OverflowTooltip.test.tsx

```diff
@@ -46,13 +48,15 @@ describe("Tooltip component", () => {
       </OverflowTooltip>
     )
 
-    expect(wrapper.props().content).toBe("")
+    const tooltip = screen.getByTestId("tooltipHoverTarget")
+    fireEvent.mouseOver(tooltip)
+
+    expect(screen.queryByText("the content")).not.toBeInTheDocument()
```

Comment:
Hmm struggling with this test because it isn't actually testing what we intend - like below we would have to use `await screen.findByText("the content")` for it to actually test that it doesn't render, but that [query](https://testing-library.com/docs/dom-testing-library/cheatsheet/#queries) throws an error w/ no match...

Reply:
Yep. I tried hovering and clicking and a bunch of other things but it just wasn't working 😓

Reply:
Looks like snapshot test necessary 😓

Reply:
Hm what do you mean by snapshot test? You mean this?
https://jestjs.io/docs/snapshot-testing

Reply:
Yup so for this first test, going to remove everything currently after the render and just have these two lines after:
```
expect(useRefSpy).toHaveBeenCalledWith(null)
expect(document.body).toMatchSnapshot()
```

Reply:
running `yarn testLib` locally will generate the snap

Reply:
done